spec/lib/tasks/deployment/20220112184331_revise_attestation_templates_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe '20220112184331_revise_attestation_templates' do
+  let(:rake_task) { Rake::Task['after_party:revise_attestation_templates'] }
+  let(:procedure) { create(:procedure) }
+  let(:attestation_template) { create(:attestation_template, procedure: procedure) }
+
+  subject(:run_task) do
+    attestation_template
+    rake_task.invoke
+    attestation_template.reload
+  end
+
+  after { rake_task.reenable }
+
+  context 'add attestation_template_id' do
+    it 'to revisions' do
```

Comment:
Je pense que c'est bien de profiter du test pour documenter un peu ce que ça fait - pour les autres gens de l'équipe, et pour les autres instances.

```suggestion
  describe 'revise_attestation_templates' do
    it 'attaches the attestation_template to the latest revision (without removing the link between attestation_template and procedure for now)' do
```